SparkPost/matchbox
# FE-794 Update Banner Component

Mockups: https://sparkpost.invisionapp.com/d/main#/console/18874652/395021232/preview

### What Changed
- Restyles the Banner Component
- Dismiss button is now a button and responds to space and enter keydown events, and is now screen reader accessible
- Status 'default' now defaults to 'info', gray banners no longer exist

<img width="848" alt="Screen Shot 2020-02-18 at 6 31 19 PM" src="https://user-images.githubusercontent.com/3903325/74787335-f058df80-527c-11ea-8420-218b0e6bc250.png">

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit the Banner stories and verify the component matches the mocks and behaves as expected

### PR Checklist
